### ISSUE TITLE
Bad ELM Response

### ISSUE THREAD

**Issue by radunre** [cedricp/ddt4all#33]
![img_20180206_113631](https://user-images.githubusercontent.com/36196149/35862726-4d445ab0-0b55-11e8-9368-dc8dfc879280.jpg)
![img_20180206_152341](https://user-images.githubusercontent.com/36196149/35862728-4d635730-0b55-11e8-9848-a89ca37b6b5e.jpg)
![img_20180206_152524](https://user-images.githubusercontent.com/36196149/35862729-4d80df30-0b55-11e8-856c-8beb391c38c5.jpg)

Hi there,

First of all, I would like to thank you so much for the idea and great work you have done for all Renault lovers.

I have started to figure out the ddt4all since 3 weeks and I have on my bench an Automatic Parking Brake unit from my Laguna 3/2010. I have connected it with an ELM Bluetooth 1.4 (as reported bu ddt4all) interface.
First issue is that ddt4all cannot find an ECU file for my unit. I have software/version D100/5A5A and database files have software/version B300 or B600/0000.
Tried and succeed to connect with the closest ECU file and it shows a lot of information.
Tried to do more, like actuating the motor and receive a strange reply, for ex. "Sending ELM request 30 18 00" followed by "Bad ELM response: WRONG RESPONSE : Unknown" 

What could be the reason for a "Bad ELM response" ?

Thank you very much in advance.

**Comment by cedricp** [comment]
I'd need your log files for analyses.

**Comment by radunre** [comment]
Here are the logs.
[log files.zip](https://github.com/cedricp/ddt4all/files/1699277/log.files.zip)

**Comment by cedricp** [comment]
Did you try to switch the diagnostic session to factory (then OK button) before ? If so, I found nothing that can help in the log files, just that the ECU refuses the sent frame...

**Comment by radunre** [comment]
Yes, I previously switched to "Factory and after sales session" that has replied fine. The "Bad ELM response" came when I tried to manually control the unit or when I tried to send movements commands.

From the screen of "02-01 - APB process" I was able to reset, start repositioning etc. but from "01-05 Manual APB command" I never succed,always 

I thought it's about a too stupid ELM but with BCM I was able to send commands without issues.

**Comment by radunre** [comment]
Sorry, button mistake on phone 😀
